### PR TITLE
[Emoji] Add the gitlab locale

### PR DESCRIPTION
**Pull request by alamirault**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no 
| Issues        | Fix #... 
| License       | MIT

Gitlab has also its own emoji shortnames

https://docs.gitlab.com/ee/development/fe_guide/emojis.html

🤐
Github: `:zipper_mouth_face:`
Gitlab: `:zipper_mouth:`

(Like lyrixx, my most important PR of the year :smiley: )

### CODE REVIEW

File: src/Symfony/Component/Emoji/Resources/data/emoji-slack.php [hunk 1]
```diff
@@ -4,6 +4,8 @@
     '👨‍❤️‍💋‍👨' => ':man-kiss-man:',
     '👩‍❤️‍💋‍👨' => ':woman-kiss-man:',
     '👩‍❤️‍💋‍👩' => ':woman-kiss-woman:',
+    '🏃‍♀️‍➡️' => ':woman_running_facing_right:',
```
Comment (alamirault) [inline src/Symfony/Component/Emoji/Resources/data/emoji-slack.php]: Updated by `make build`, should I revert slack emojis ?

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/Emoji/Resources/data/emoji-slack.php (hunk 1 above)
Comment (stof) [reply src/Symfony/Component/Emoji/Resources/data/emoji-slack.php]: I would suggest doing first a separate PR submitting the updated Slack data (by running the build without your new code handling gitlab) and then rebasing your PR (or redoing the build entirely) once the slack update is merged

**Review by stof — COMMENTED** [review]
Support for `gitlab` should also be added in the `emojify` Twig filter

File: src/Symfony/Component/Emoji/Resources/data/emoji-slack.php (hunk 1 above)
Comment (alamirault) [reply src/Symfony/Component/Emoji/Resources/data/emoji-slack.php]: I created https://github.com/symfony/symfony/pull/54450

**Comment by alamirault** [comment]
> Support for `gitlab` should also be added in the `emojify` Twig filter

With actual PR https://github.com/symfony/symfony/pull/54432, no change is required in filter to support new "locale"

**Review by stof — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @alamirault.

**Pull request merged**